src/libData/AccountData/Account.h

```diff
@@ -43,29 +44,39 @@ class Account : public Serializable
 {
     boost::multiprecision::uint256_t m_balance;
     boost::multiprecision::uint256_t m_nonce;
-    h256 m_storageRoot = h256();
-    h256 m_prevRoot;
+    h256 m_storageRoot, m_prevRoot;
```

Comment:
Why do we need a m_prevRoot exactly?

Reply:
That's necessary for persistence recovery purpose at current design, it won't be written into disk. Let's keep it first.

Reply:
ok